ffi/dart/test/icu_test.dart

```diff
@@ -148,6 +152,29 @@ void main() {
       'Mittwoch, 15. Januar 2025, 14:32',
     );
 
+    ///// ZonedDateFormatter /////
```

Comment:
nit: this is a 200 line test with comments splitting it into different areas. just split the test